.github/workflows/integrate.yml

```diff
@@ -0,0 +1,71 @@
+name: Run PHPUnit and PHPCS
+
+on: [push]
```

Comment:
See https://github.com/Parsely/wp-parsely/blob/develop/.github/workflows/integration-tests.yml#L4-L11 - in this case, we:

- can avoid running everything when there is just a `.md` file update (i.e. update Changelog, or docs)
- include running on PRs, since if someone forks and opens a PR, this current workflow won't run.
- enable manual running of workflows.